sisyphus/son_of_grid_engine.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, default_rqmt, gateway=None, auto_clean_eqw=True, ignore_jobs=
         :param list[str] ignore_jobs: list of job ids that will be ignored during status updates.
                                       Useful if a job is stuck inside of SGE and can not be deleted.
                                       Job should be listed as "job_number.task_id" e.g.: ['123.1', '123.2', '125.1']
+        :param str pe_name: used for parallel environment (PE), when parallel_tasks is set to rqmt
```

Comment:
```suggestion
        :param str pe_name: used to select parallel environment (PE), when parallel_tasks is set in rqmt
```
suggest improved wording